libs/community/tests/integration_tests/retrievers/test_azure_ai_search.py

```diff
@@ -68,3 +68,32 @@ async def test_azure_cognitive_search_ainvoke() -> None:
     for doc in documents:
         assert isinstance(doc, Document)
         assert doc.page_content
+
+
+def test_azure_ai_search_invoke_managed_identity() -> None:
```

Comment:
So all of the tests are actually testing with real credentials, versus using mocks?

Reply:
Yes, the person running the tests has to add in their own. I'm following what the rest of the tests went with